setup.cfg

```diff
@@ -0,0 +1,11 @@
+[flake8]
```

Comment:
Mypy black configs can go to a single pyproject. toml. Typically this is the file with all configs.  As far as I know flake8 does not support pyproject.toml, so let this file be here, but we will not use flake in the future tasks anyway

Reply:
Allright, but I decided to replace configurations for flake8 from setup.cfg to .flake8. I'm just experimenting =)